src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -872,19 +907,17 @@ private static bool TryGetEasyRequestFromContext(IntPtr context, out EasyRequest
                 return false;
             }
 
-            [Conditional(VerboseDebuggingConditional)]
-            private void VerboseTrace(string text = null, [CallerMemberName] string memberName = null, EasyRequest easy = null)
+            private void EventSourceTrace<TArg0>(string formatMessage, TArg0 arg0, EasyRequest easy = null, [CallerMemberName] string memberName = null)
```

Comment:
Is it possible to not have these methods duplicated? Since these are all nested classes, can these methods be moved to the outer CurlHandler class so they can be shared?